### ISSUE TITLE
ReachWeb Advertising or District0x Web Advertising

### ISSUE THREAD

**Issue by PerryEdu** [district0x/district-proposals#266]
**Name**
ReachWeb Advertising or District0x Web Advertising

**Purpose**
The purpose of ReachWeb will be to enable district marketplaces to advertise, generate traffic and increase exposure for the district. It will also provide a platform for any other companies or individual to advertise.

**Description**
_Advertising_
The site will feature two types of advertising: _Classified ads_ and _articles_.

_Training videos_ to explain how to most effectively advertise on the web will be included. For example, how to determine high search volume keywords, use of bolded words that link to external content (other articles, web-content, statistical information, advertiser websites), the importance of titles, and use of URLs to mention a few. 

The _training videos_ could additionally include ideas for other types of web adverting, for example, AdWords, social media marketing, Twitter, Linkedin, Twitter, blogs, Facebook, Youtube videos, email marketing, forums, webinars. In other words, the site would not only provide access to advertise, it would provide added-value content that would assist marketplace owners to promote their offering and learn about web marketing.

The site would offer different categories and subcategories to place either classifieds or articles (two separate sections). The site would need to be updated from time to time to add categories **or** categories/subcategories could be added by the user. 

The site would include templates and tools to creatively design engaging advertising content (with more training videos), including the embedding of video content.

Adverts would be automatically removed, for instance, every three months. The advertiser will receive a notification a week before removal to allow for time to re-insert (one click) if required. This would create space for adverts no longer needed. _Alternately, advertisers could have the option to insert the advert or article for one, two or three months with the option to re-insert with one click one week before the expiry date._

An additional tab could provide contact information for professional services that specialize in internet marketing.  In other words, many advertisers will do their own and many other others who have a budget may do their own and get some professional help.

**Payments**
Users will pay a nominal registration fee to join the district, for example, $10  paid every 6 or 12 months and would allow the user unlimited access to the advertising platform. Advertisers would be reminded by email three weeks before of renewal or it could be automated unless canceled (one click). The fee could be shared as follows (as a suggestion): 10% for the originator, 30% Dictrict0X team, 30% by the community members who initially vote for the district (shared equally), and 30% for rewards/voting for changes additions, cost of space etc.
Similar adverting sites charge monthly fees. 

**Conclusion**
In conclusion, any marketplace to be successful needs to have an advertising campaign.  Any district that markets their offering will generate traffic and increase exposure directly to their business and indirectly to the entire District0x project.  Furthermore, such a platform could attract huge interest that would result in increased growth of District0x districts.

**Ethereum Address:**
0x6be57656C371FA67f37E50EE2D9Ad5cdCDd2b882
